api/graphql/index.js

```diff
@@ -126,6 +127,7 @@ function createSchema (userId, isAdmin) {
       resendInvitation: (root, {invitationId}) => resendInvitation(userId, invitationId),
       reinviteAll: (root, {communityId}) => reinviteAll(userId, communityId),
       flagInappropriateContent: (root, { data }) => flagInappropriateContent(userId, data),
+      removePost: (root, { postId, slug }) => removePost(userId, postId, slug),
```

Comment:
here you could do `({communityId, postId, slug}) => removePost(userId, postId, communityId || slug)`